routes/users.php

```diff
@@ -2,21 +2,21 @@
 
 // Users
 $router->group([
-    'prefix' => 'users',
     'middleware' => ['scope:users']
 ], function () use ($router) {
     // Public access
-    $router->get('/', 'UsersController@index');
-    $router->get('/{id:[0-9]+}', 'UsersController@show');
+    resource($router, 'users', 'UsersController', [
+        'only' => ['index', 'show']
+    ]);
 
     // Restricted access
     $router->group([
         'middleware' => ['auth:api', 'scope:users']
     ], function () use ($router) {
-        $router->post('/', 'UsersController@store');
-        $router->put('/{id:[0-9]+}', 'UsersController@update');
-        $router->delete('/{id:[0-9]+}', 'UsersController@destroy');
-        $router->get('/me', 'UsersController@showMe');
-        $router->put('/me', 'UsersController@updateMe');
+        resource($router, 'users', 'UsersController', [
+            'only' => ['store', 'update', 'destroy']
+        ]);
+        $router->get('/users/me', 'UsersController@showMe');
```

Comment:
There are a lot of minor exceptions like these. I do wonder how useful the helper is if I'm constantly adding extra routes.